src/app/components/client/ExposureCard.tsx

```diff
@@ -57,7 +57,7 @@ export type ExposureCardProps = {
 };
 
 type DetailsFoundProps = {
-  whichExposed: string; // family | email | phone | address | credit card | password
+  whichExposed: string; // family | email | phone | address | creditcard | password
```

Comment:
Instead of generic "string" should we inline the valid values?
```suggestion
  whichExposed: "family" | "email" | "phone" | "address" | "creditcard" | "password";
```